packages/terra-core-docs/src/terra-dev-site/doc/toggle-button/example/OpenCloseEventToggleButton.module.scss

```diff
@@ -2,4 +2,11 @@
   .times-wrapper {
     display: inline-block;
   }
+
+  .toggle-button-content {
+    a:focus {
+      outline: 2px dashed #000;
+      outline-offset: 2px;
+    }
```

Comment:
since it is common style used for multiple examples can we move this styles to togglebuttoncommon style sheet instead of having it on individual style sheets

Reply:
will do it. Thanks